packages/html-reporter/src/headerView.tsx

```diff
@@ -135,24 +143,24 @@ const SettingsButton: React.FC = () => {
       }}
       onMouseDown={preventDefault}>
       {icons.settings()}
+      <Dialog
```

Comment:
moving the dialog up is necessary, otherwise it becomes the last-child and the styling breaks.

https://github.com/microsoft/playwright/blob/21bb265aa1f290b297cc7c9c78596a785216b667/packages/html-reporter/src/common.css#L223